L/libmathgl/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libmathgl"
+version = v"0.0.1"
```

Comment:
Shouldn't this be 2.4.4?  See https://docs.binarybuilder.org/stable/building/#Version-number